registration/register.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+
+  <!--Add the title of your typeform below-->
+  <title>Registration Form</title>
+
+  <!--CSS styles that ensure your typeform takes up all the available screen space (DO NOT EDIT!)-->
+  <style type="text/css">
+    html{
+      margin: 0;
+      height: 100%;
+      overflow: hidden;
+    }
+    iframe{
+      position: absolute;
+      left:0;
+      right:0;
+      bottom:0;
+      top:0;
+      border:0;
+    }
+  </style>
```

Comment:
Let's move this css into a separate file:
/registration/frontend/main.css

And then link the css file in the head:
http://www.w3schools.com/tags/tag_link.asp